airlock/urls.py

```diff
@@ -124,10 +124,15 @@
         name="group_edit",
     ),
     path(
-        "requests/comment/<str:request_id>/<str:group>",
+        "requests/comment/create/<str:request_id>/<str:group>",
```

Comment:
nb. I've slightly cut a corner here but revising this url but leaving all the methods as `group_comment` instead of updating to `group_comment_create` - should I do this?

Reply:
changed